patterns/core/sass/settings/_colors.scss

```diff
@@ -0,0 +1,7 @@
+$ColorScheme: (
```

Comment:
If everything within this lib is from north, that needs to be mentioned and linked somewhere, but I think we need to specify the documentation on this for Watson and our instance of `ColorScheme`, etc.